modules/aws/ecs.go

```diff
@@ -90,6 +90,56 @@ func DeleteEcsClusterE(t *testing.T, region string, cluster *ecs.Cluster) error
 	return err
 }
 
+// GetEcsService fetches information about specified ECS service.
+func GetEcsService(t *testing.T, region, clusterName, serviceName string) *ecs.Service {
```

Comment:
We actually like to set the type on each variable (e.g `region string, clusterName string, serviceName string`), even if they are the same. It is more verbose, but we find that it reads better. Can you update?

Reply:
Done.